disco_aws_automation/disco_metanetwork.py

```diff
@@ -412,7 +412,8 @@ def update_gateways_and_routes(self, desired_route_tuples, dry_run=False):
         current_route_tuples = set()
         if self.centralized_route_table:
             for route in self.centralized_route_table.routes:
-                if route.gateway_id and route.gateway_id != 'local':
+                if route.destination_cidr_block and \
```

Comment:
We use current_route_tuples to figure out what is missing in the disco_vpc.ini, and we would delete the routes that are no longer there. We don't want to include the S3 endpoint routes or any routes that don't have a CIDR block here because we do not manage those.